code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -996,13 +995,20 @@ func (srh *StorageRestHandler) getOpenChallenges(w http.ResponseWriter, r *http.
 	if edb == nil {
 		common.Respond(w, r, nil, common.NewErrInternal("no db connection"))
 	}
-	blobber, err := edb.GetBlobber(blobberID)
+
+	conf, err := getConfig(sctx)
+	if err != nil && err != util.ErrValueNotPresent {
```

Comment:
Let's not hit the MPT to get config. We can get value from config.MaxChallengeCompletionTime for now. Ideally we will load the config from MPT on node starts and update it on each block finalization. So then we will only access config from the in-memory config to avoid MPT accessing.

Reply:
fixed